Consider tabindex in iframe has acc name (#1102)
act-rules/act-rules.github.io: _rules/iframe-accessible-name-cae760.md

## Assumptions

> If an `iframe` is not perceived by the user as a single control, it does not qualify as a [user interface component][] under WCAG 2. In such a scenario, failing this rule would not fail [success criterion 4.1.2](https://www.w3.org/TR/WCAG21/#name-role-value). Unless the `iframe` is both removed from the accessibility tree and removed from [sequential focus navigation][], they usually are considered to be [user interface components][user interface component].

Reviewer: I had the impression that we usually start the assumptions by "this rule assumes that".
Not a big deal, but uniformity helps.

## Accessibility Support

> Some browsers include `iframe` elements in the [sequential focus navigation][]. This ensures that `iframe` elements can always be scrolled using the keyboard. When an `iframe` is removed from the accessibility tree, this rule is still applicable for those browsers, unless the `tabindex` attribute is set to `-1`.

Reviewer: ```suggestion
Some browsers include `iframe` elements in the [sequential focus navigation][]. This ensures that `iframe` elements can always be scrolled using the keyboard. When an `iframe` is removed from the accessibility tree, this rule is still applicable for those browsers, unless the `iframe` is explicitly removed from [sequential focus navigation][] (by having the `tabindex` attribute set to a negative value).
```
Because `tabindex=-2` also works…

#### Failed Example 7

> This `iframe` that is part of [sequential focus navigation][] has an empty [Accessible name][].

Reviewer: We need a similar Passed Example (not in the accessibility tree, in sequential focus order, has an accessible name).

Author: I don't think that's necessary.

#### Inapplicable Example 2

> This `iframe` is not part of [sequential focus navigation][], nor [included in the accessibility tree][] because of `display: none;`.

Reviewer: ```suggestion
This `iframe` is neither part of [sequential focus navigation][], nor [included in the accessibility tree][] because of `display: none;`.
```

> This `iframe` is not part of [sequential focus navigation][], nor [included in the accessibility tree][] because of `display: none;`.

Reviewer: As far as I understand, browsers that "force push" iframes in sequential navigation won't do it here because of the `display:none`, which is fine.
The description, however, looks a bit like the "because of `display:none` is related only to the second part (not in the accessibility tree) rather than to both.

But I don't really have any idea to improve it…

Author: OK, leaving it as is, since it got 2 approvals.

```html
<iframe style="display:none;" src="/test-assets/SC4-1-2-frame-doc.html"> </iframe>
```

#### Inapplicable Example 3

> This `iframe` is not part of [sequential focus navigation][] because it has `tabindex="-1"` and not [included in the accessibility tree][] because of `role="presentation"`

Reviewer: ```suggestion
This `iframe` is neither part of [sequential focus navigation][] (because it has `tabindex="-1"`) nor ([included in the accessibility tree][] because of `role="presentation"`).
```

Author: I don't know what this is for, seems unnecessary.
